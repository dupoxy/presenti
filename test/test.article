# DO NOT EDIT GENERATED by presenti
# from source line 1
A presenti test file

The presenti Authors
https://www.github.com/dupoxy/presenti/

# from source line 15
* Title of slide or section (must have asterisk)

Some Text

** Subsection (slide does not support Subsection)

- bullets
- more bullets
- a bullet with

*** Sub-subsection

Some More text

  Preformatted text
  is indented (however you like)

Further Text, including invocations like:

.code test.go /^func main/,/^}/

The above code should have `func main` line highlighted.

.code test.go /^func main/,/^}/ HL001

The above code should have `fmt.Println` line highlighted.

.code -edit test.go /^func main/,/^}/

The above code should have `-edit` activated.

.code -numbers test.go /^func main/,/^}/

The above code should have `-numbers` activated.

.play test.go /^func main/,/^}/

The above code should have `func main` line highlighted and be playable.

.play test.go /^func main/,/^}/ HL001

The above code should have `fmt.Println` line highlighted and be playable.

.play -edit test.go /^func main/,/^}/

The above code should have `-edit` activated and be playable.

.play -numbers test.go /^func main/,/^}/

The above code should have `-numbers` activated and be playable.

# This should return an error ? .html, .code and .play does.
.image image.jpg

The above image should return an error ? .html, .code and .play does.

.iframe http://foo
.link http://foo label
.caption _Gopher_ by [[http://www.reneefrench.com][Ren√©e French]]

Again, more text

# from source line 88
EOF comment Test.
