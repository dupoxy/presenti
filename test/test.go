/* #nb:
A Good presenti Example

The presenti Authors
https://www.github.com/dupoxy/presenti/
*/

// A presenti test file.
package main

import (
	"fmt"
)

/* #nb:
* Title of slide or section (must have asterisk)
Some Text
** Subsection
- bullets
- more bullets
- a bullet with
*** Sub-subsection
Some More text
	Preformatted text
	is indented (however you like)
Further Text, including invocations like:
.code test.go /^func main/,/^}/
.play test.go
# .image image.jpg
# .iframe http://foo
*/

/* #nb: Their IS NO new Line at end of comment
 */
func main() {

	/* #nb: THIS IS NOT a top level comment
	Bla bla */

	fmt.Println("good")
}
/* #nb: EOF comment Test. */
