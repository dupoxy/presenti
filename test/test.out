# DO NOT EDIT GENERATED by presenti
A Good presenti Example

The presenti Authors
https://www.github.com/dupoxy/presenti/

* Title of slide or section (must have asterisk)
Some Text
** Subsection
- bullets
- more bullets
- a bullet with
*** Sub-subsection
Some More text
	Preformatted text
	is indented (however you like)
Further Text, including invocations like:
.code test.go /^func main/,/^}/
.play test.go
# .image image.jpg
# .iframe http://foo

EOF comment Test.
